src/Components/Featured/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { data } from "./data";
+import MediaCard from "../MediaCard";
+import styles from "./style.module.css";
+import { Box } from "@mui/material";
+
+export default function Featured() {
+  return (
+    <Box className={styles.main_box}>
+      {data.map((item, index) => (
+        <Box key={index} className={styles.featured_box}>
```

Comment:
You should not use the index from the map method as a key for the lists. [See why in the Pitfall section](https://react.dev/learn/rendering-lists#why-does-react-need-keys).
Please use a better unique key.